rules/windows/powershell/powershell_script/posh_ps_etw_trace_evasion.yml

```diff
@@ -1,4 +1,4 @@
-title: Disable of ETW Trace
+title: Disable of ETW Trace - Powershell Script Block Text
```

Comment:
`title: Disable of ETW Trace - PowerShell`
Like the other rules

Reply:
Modified